frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -762,6 +764,24 @@ const isKindMonitorable = (model: K8sKind): boolean => {
   }
 };
 
+export const getWorkloadMonitoringAlerts = (
+  monitoringAlerts: Alerts,
+  resource: K8sResourceKind,
+): Alert[] => {
```

Comment:
Could you switch the parameter order to be more consistent with other functions in resource-utils?